### PR TITLE
check for openapi or swagger key in swagger object

### LINKED ISSUE #31: swagger = require('dereferenced-swagger-file'); this module is not present in node

### PR DESCRIPTION
**Pull request by gt-novelt**
fixes #31

### CODE REVIEW

File: lib/validate-swagger.js [hunk 1]
```diff
@@ -21,8 +21,13 @@ function validateSwagger (swaggerDef) {
  * @param  {Object} swaggerDef - dereferrenced swagger object
  */
 function isSwagger (obj) {
-  var requiredKeys = ['swagger', 'info', 'paths']
+  var requiredKeys = [['openapi', 'swagger'], 'info', 'paths']
```
Comment (CacheControl) [inline lib/validate-swagger.js]: @gt-novelt how about we just remove 'swagger' from this list altogether?  This method was never meant to be an full fledged validator anyway; just a sanity check for developers who accidentally pass invalid data.   The library doesn't even use the "swagger" key, so I think we're good just changing this array to contain only `info` and `paths`

**Comment by CacheControl** [comment]
Closing due to lack of activity

File: lib/validate-swagger.js (hunk 1 above)
Comment (gt-novelt) [reply lib/validate-swagger.js]: @CacheControl sry for the late reply.

Yes removing the key sounds reasonable. Even openAPI does not rely on it.

Comment (gt-novelt) [reply lib/validate-swagger.js]: Actually double checking the openAPI specs, the key is *REQUIRED*

> openapi | string | REQUIRED. This string MUST be the semantic version number of the OpenAPI Specification version that the OpenAPI document uses. The openapi field SHOULD be used by tooling specifications and clients to interpret the OpenAPI document. This is not related to the API info.version string.

src : https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#fixed-fields

Comment (CacheControl) [reply lib/validate-swagger.js]: I think that although the spec requires it, `hippie-swagger` does not; it's silly for this library to validate a field it doesn't even use.  So I think I'm still leaning towards keeping things simple and removing `swagger` from this list.

**Pull request closed without merging**